JS/orm.js

```diff
@@ -27,6 +27,7 @@ UserAnswers.init(
   {
     id:{
       primaryKey : true,
+      autoIncrement: true,
       type: Sequelize.INTEGER,
     },
```

Comment:
Turns out our whole config for `id` is just the default, including what you added, so let's remove this entire block and it will work correctly.

Reply:
Removed the autoincrement value, should I remove the whole `id` field definition?